etcdserver/verify/doc.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The etcd Authors
+// Copyright 2024 The etcd Authors
```

Comment:
nit: I think we should keep the original date here.

Reply:
Hey @ivanvc. I'll make that change. Is it OK to leave the other dates as 2024?

Reply:
Yes, but I don't see the need to update the date in this file.